platform_plugin_elm_credentials/api/views.py

```diff
@@ -87,26 +98,49 @@ def get(self, request, course_id: str) -> HttpResponse:
                 status_code=status.HTTP_404_NOT_FOUND,
             )
 
-        user = request.user
-        certificate = GeneratedCertificate.certificate_for_student(user, course_id)
+        user_has_access = any(
+            [
+                request.user.is_staff,
+                CourseStaffRole(course_key).has_user(request.user),
+                CourseInstructorRole(course_key).has_user(request.user),
+            ]
+        )
 
-        if not certificate:
+        if not user_has_access:
```

Comment:
I thought we were allowing students to generate their credentials as well. Did that change?